src/af-confidence/behaviour.tsx

```diff
@@ -0,0 +1,99 @@
+// from https://github.com/PDBeurope/pdbe-molstar/tree/master/src/app/af-confidence
```

Comment:
The referenced code has the (thankfully permissive) Apache 2.0 license, which means you have to attribute the code within the license terms:

> If you redistribute software with any Apache licensed components, you must include a copy of the license, provide a clear Apache License attribution, and add modification notices to all the files that you modify. [source](https://www.whitesourcesoftware.com/resources/blog/top-10-apache-license-questions-answered/#:~:text=of%20the%C2%A0license%2C-,provide,-a%20clear%20Apache)

My understanding is (and @lgruen will be able to confirm) to include these files (including below), you will have to include a header in this file to say: _<Apache 2.0 header>, I have / have not made these specific modifications_.

Reply:
Will add this license once confirmed. Where do I find the Apache 2.0 header?

Reply:
The general header is here: https://www.apache.org/licenses/LICENSE-2.0#apply, make sure to fill in the fields from the original header and add something like "Original file reproduced without modification from <link to specific commit>" (rather than the `master` branch you're referencing).